src/vs/code/electron-sandbox/workbench/workbench.html

```diff
@@ -3,7 +3,7 @@
 <html>
 	<head>
 		<meta charset="utf-8" />
-		<meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' https: data: blob: vscode-remote-resource:; media-src 'none'; frame-src 'self' vscode-webview:; object-src 'self'; script-src 'self' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline'; connect-src 'self' https: ws:; font-src 'self' https: vscode-remote-resource:;">
+		<meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' https: data: blob: vscode-remote-resource:; media-src 'self'; frame-src 'self' vscode-webview:; object-src 'self'; script-src 'self' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline'; connect-src 'self' https: ws:; font-src 'self' https: vscode-remote-resource:;">
```

Comment:
Touches `Content-Security-Policy`.
fyi @bpasero @jrieken

Reply:
fyi - there is a few more places where CSP is defined, e.g the web, vscode.dev. github.dev etc ppp

Reply:
How should we proceed here?

Reply:
@bpasero @deepak1556 @jrieken @alexdima  do you have some security concerns with changing `media-src` in the `Content-Security-Policy`

Reply:
Probably needs tracking adoption items in any host of VSCode web.